Tests/TuistCoreTests/Graph/GraphTraverserTests.swift

```diff
@@ -4731,6 +4731,58 @@ final class GraphTraverserTests: TuistUnitTestCase {
         XCTAssertEqual(got, Set([GraphTarget(path: packageProject.path, target: packageDevProduct, project: packageProject)]))
     }
 
+    func test_allOrphanRemoteTargets_whenLocalPackageHasOrphanTargets() throws {
+        // Given
+        let app = Target.test(name: "App", product: .app)
+        let project = Project.test(path: try! AbsolutePath(validating: "/App"), targets: [app])
```

Comment:
```suggestion
        let project = Project.test(path: "/App", targets: [app])
```

nit: I think you can use a string literal instead of the throwing `AbsolutePath` init. If not, then we can at least change `try!` to `try` as this method is throwing anyway. The same goes for the rest of the changes (there are a couple of similar instances)

Reply:
Updated the new tests